### ISSUE TITLE
Conduct Nov 1st–3rd compensation request voting

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/compensation#149]
@ripcurlx commented on [Tue Oct 30 2018](https://github.com/bisq-network/bisq/issues/1843)

_(previous voting round: #121 )_

 - [x] Announce call for requests 3 days ahead and provide instructions on how to submit in a comment on this issue
 - [ ] Transcribe [compensation requests](https://github.com/bisq-network/compensation/issues) to [stake and voting spreadsheet](https://docs.google.com/spreadsheets/d/1xlXDswj3251BPCOcII-UyWlX7o7jMkfYBE-IZ5te5Ck/edit#gid=912569327)
 - [ ] Announce that the voting period has begun in a comment on this issue (also: created Slack reminder)
 - [ ] Wait for the voting period to complete
 - [ ] Review all edits to the spreadsheet to ensure there weren't any mistakes, etc
 - [ ] Protect this month's spreadsheet worksheet such that only the @bisq-network/compensation-maintainers can edit it
 - [ ] Announce that the voting period is complete in a comment below (see prior months' voting epics for examples)
 - [ ] Close all related compensation request issues with a comment  (see prior months' compensation requests for examples)
 - [ ] Complete any tasks in the comments below

---

@ripcurlx commented on [Tue Oct 30 2018](https://github.com/bisq-network/bisq/issues/1843#issuecomment-434213765)

@bisq-network/contributors, please submit any compensation requests by EOD on Wednesday October 31st.

As usual, instructions can be found at https://docs.bisq.network/dao/phase-zero.html#how-to-request-compensation. Thanks.

---

@ripcurlx commented on [Tue Oct 30 2018](https://github.com/bisq-network/bisq/issues/1843#issuecomment-434219198)

I missed a duplication for @flix1 BSQ address in the voting spreadsheet. I just merged the two entries and deleted the duplicate entry.

**Comment by ripcurlx** [comment]
@bisq-network/contributors, please submit any compensation requests by EOD on Wednesday October 31st.

As usual, instructions can be found at https://docs.bisq.network/dao/phase-zero.html#how-to-request-compensation. Thanks.

**Comment by ripcurlx** [comment]
I missed a duplication for @flix1 BSQ address in the voting spreadsheet. I just merged the two entries and deleted the duplicate entry.

**Comment by ripcurlx** [comment]
Sorry about the recreation of the voting epic. I use ZenHub to manage my GitHub repositories locally and it changed the default repository when creating new issues/epics.

**Comment by ripcurlx** [comment]
I just changed @flix1 Mainnet address from `B1Pdq17bL5GiitCsNybi3Fc3V8qa7btYnPe` to `B18pxrZRFJ7Q1PsND9xqayk1uQbtZiH39MC` as he lost access when he updated to v0.8.0.

**Comment by ripcurlx** [comment]
Compensation requests have been staged in the BSQ stake and vote tracking spreadsheet, and the voting period is now open.

The specific worksheet for this month is at https://docs.google.com/spreadsheets/d/1xlXDswj3251BPCOcII-UyWlX7o7jMkfYBE-IZ5te5Ck/edit#gid=1237123520

Voting instructions are at https://docs.bisq.network/dao/phase-zero.html#how-to-vote.

Some of you will be voting for the first time and will need access. Just request it and I'll grant it.

Please don't forget that we are voting this time again based on the [Natural delegation on DAO voting proposal](https://github.com/bisq-network/proposals/issues/34). Only vote `+1` on compensation requests you can evaluate and do know about, for all other compensation requests vote `0` to show your intention to delegate your vote.

Please complete your voting by EOD on Saturday November 3rd.

Thanks all.

**Comment by cbeams** [comment]
Weren't we going to introduce a 3-day review period prior to voting, per https://github.com/bisq-network/proposals/issues/40?

That proposal got into details about how to implement it in the DAO code, but I was always under the impression that we'd implement it in the manual, spreadsheet-based version as well.

Shall we do so starting next month?

**Comment by ripcurlx** [comment]
Makes sense to do it also for the manual voting 👍 . So we'll have for next month's compensation requests a 3-day review period (1st-3rd) followed by a 3-day voting period (4th-6th).

**Comment by sqrrm** [comment]
@cbeams @ripcurlx There was support for making the break longer but it seems one or two days as suggested by Manfred would be the conclusion after all the rather confusing discussion. I think Manfred is now setting the BREAK1 phase to 150 blocks for the testnet release so perhaps we start with a one day break and do voting on the 2nd-4th.

**Comment by ripcurlx** [comment]
@sqrrm Thanks for pointing this out. I'll adapt the periods accordingly.

**Comment by HarryMacfinned** [comment]
Welcome indeed to the new contributors. :+1: 
Happy to see you.
